usage/camera/raspicam/raw.md

```diff
@@ -0,0 +1,27 @@
+# Using RAW on the Raspberry pi cameras
+
+The definition of raw images can vary. The usual meaning is raw bayer data directly from the sensor, although more unusually, some may regard an uncompressed image that has passed through the ISP (and has therefore been processed) as raw.
```

Comment:
Given that the sentence starts with "The usual meaning ...", perhaps "..., although more unusually, some may ..." could be rewritten as "..., although some may ..." ?